roles/contiv_network/tasks/main.yml

```diff
@@ -82,8 +85,12 @@
   copy: src=netmaster.service dest=/etc/systemd/system/netmaster.service
   when: run_as == "master"
 
+- name: reload system config for netmaster
+  shell: systemctl daemon-reload
+  when: run_as == "master"
```

Comment:
Sure, I can make that change as well. I wasn't really adding or changing the daemon-reload option, so I didn't check etcd and the other services. I will check and add that change along with this.